http_check/assets/configuration/spec.yaml

```
@@ -10,7 +10,10 @@ files:
     options:
     - name: name
       required: true
-      description: Name of your Http check instance.
+      description: |
+        Name of your Http check instance.
```

nit - we have `HTTP` (uppercase) everywhere else in this file

```suggestion
        Name of your HTTP check instance.
```